test/srt/lora/test_lora_normalization.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023-2024 SGLang Team
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
IMHO, the change is too trivial to add a new unit test, I would probably suggest either:
1. find an adapter with this pattern (e.g., llama4) and add it to our existing lora test, or
2. I am okay with checking in your change without additional tests as long as it passes existing ones.

Reply:
More context: we are actually cleaning up low-value UT to speed up our CI run